bedrock/mozorg/templates/mozorg/contribute/index.html

```diff
@@ -1,130 +1,313 @@
+{{ css_bundle('contribute-base') }}
 {% endblock %}
 
-{% block body_id %}landing{% endblock %}
-
-{% block contrib_nav_cta %}{% endblock %}
-
-{% block contrib_page_title %}{{ _('Volunteer with Mozilla!') }}{% endblock %}
-
-{% block contrib_content %}
-  {% call hero(
-    title=_('Join a global community of game-changers.'),
-    class='mzp-t-dark contribute-hero',
-    include_cta=True,
-    heading_level=2
-  ) %}
-
-    <a href="https://activate.mozilla.community/?utm_source=www.mozilla.org&amp;utm_medium=referral&amp;utm_campaign=contribute&amp;utm_content=contribute-page" class="mzp-c-button mzp-t-dark" data-link-type="button" data-link-name="Get Involved" data-cta-position="Primary">{{ _('Get Involved') }}</a>
-  {% endcall %}
-
-  <div class="mzp-l-content">
-    {% call feature_card(
-      title=_('Volunteering with Mozilla&hellip;'),
-      image_url='img/contribute/video-image.png',
-      aspect_ratio='mzp-has-aspect-16-9',
-      class='mzp-l-card-feature-left-half'
-    ) %}
-      {# L10n: This paragraph follows the headline, "Volunteering with Mozilla...", so the first sentence continues from that fragment. #}
-      {% trans %}
-        is about more than lending a hand. It’s about learning, collaborating
-        and sharing your skills with a global community of Mozillians — coders,
-        organizers, activists and more — who help shape the Web every day.
-      {% endtrans %}
-      </p>
-      <p class="mission-cta">
-        <a class="mzp-c-cta-link js-contribute-video" href="https://videos.cdn.mozilla.net/uploads/mozillaorg/Mozilla_2014_i_am.webm">
-          {{ _('See our community in action') }}
+
+{% block content %}
+<main role="main">
+  <div class="mzp-l-content contribute-intro">
+    <header class="contribute-intro-content">
+      <h1 class="contribute-intro-heading">{{ _('Your talents are needed for
+        a
+        better internet') }}</h1>
+      <p>{{ _('Mozilla is a non-profit organization working to ensure the
+        internet
+        is open and welcoming to all. And we need your help. By joining our
+        community, you can help drive innovation, enhance accountability and
+        trust
+        and make the internet a better place for everyone.') }}</p>
+    </header>
+
+    <div class="contribute-intro-media-wrapper">
+      <div class="contribute-intro-media">
+        <img src="/media/img/contribute/contribute-hero-composition.jpg"
+          alt="">
+      </div>
+    </div>
+  </div>
+
+  <section class="contribute-hero">
+    <div class="mzp-l-content">
+      <h2>{{ _('How you can contribute&hellip;') }}</h2>
+    </div>
+    <div class="mzp-l-content mzp-l-card-quarter
+      contribute-card-container">
+      <section class="mzp-c-card contribute-card">
+        <a class="mzp-c-card-block-link contribute-card-wrapper"
+          href="https://community.mozilla.org/activities/localize-mozilla">
+          <div class="mzp-c-card-media-wrapper">
+            <picture>
```

Comment:
I'm not sure if this will work - the asset is based on screen size, not resolution.
I just attempted to use this and couldn't get the right image asset across viewport sizes.

Reply:
Ah I see... in that case we may still need to do it this way, but just switch to the `static()` helper for the image URLs.

Reply:
Updated in latest changes!